rootfs-version-check/README.md

```diff
@@ -0,0 +1,48 @@
+## Description
+
+The rootfs-version-check Update Module implements a full image update with additional checks to protect against replay attacks.
+
+This is functionally equivalent to the built-in full image update with an extra check to ensure the artifact name follows a specific
+format and that installing "older" images is rejected.  For this reference implementaton, we simply use a numeric identifier and ensure
+that it is larger than the version installed.  For actual device fleet use, you may need to customize this based on your artifact
+naming scheme.
```

Comment:
I think that the `mender-compare-versions` script belongs to this repo (as the description here seems to indicate). I would move it here and rework the meta-mender-community recipe accordingly.

Reply:
I'm not sure that's appropriate.  This is really an integration point.  Do we really want it here?

Reply:
To me is part of the update module, while meta-mender-comminity is just the recipe on how to install it (I could install it myself just copying files, right? No strong opinion in any case, you have more experience in yocto best practices

Reply:
Yeah, I get that argument and I don't have a strong opinion either.  @mirzak ?

Reply:
Makes sense to keep it here, at least as a reference script that people can override in e.g Yocto

Reply:
OK.  I moved it to this repo and updated both PRs.  This is ready to go I believe.